### DISCUSSION TITLE
Extract code for organising

### DISCUSSION THREAD

**Discussion by evertondanilo** [radix-ui/themes#36] [Q&A] [answered]
```
<Tabs
   <Tabs.Trigger
```

Let's say I want to create a component that gives back <Tabs.Trigger... can I do that? How?

```
<Tabs
   <AcomponentthatReturnsTabTrigger 
```

I was trying something similar and I got an error saying unsuitable for render

**Comment by andy-hook** [comment, marked as answer]
You can return the part after forwarding `MyTrigger` props and refs to it:

```jsx
const MyTrigger = React.forwardRef((props, forwardedRef) => {
  return <Tabs.Trigger {...props} ref={forwardedRef} />;
});
```
